.github/workflows/cpflow-deploy-staging.yml

```
@@ -0,0 +1,123 @@
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup environment
+        uses: ./.github/actions/cpflow-setup-environment
+        with:
+          token: ${{ secrets.CPLN_TOKEN_STAGING }}
+          org: ${{ vars.CPLN_ORG_STAGING }}
+
+      - name: Build Docker image
+        uses: ./.github/actions/cpflow-build-docker-image
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/cpflow-build-docker-image": yaml: line 62: could not find expected ':'

(action)